tapiriik/sync/sync.py

```diff
@@ -788,6 +788,7 @@ def _downloadActivityList(self, conn, exhaustive, no_add=False):
 
         self._accumulateExclusions(conn, svcExclusions)
         self._accumulateActivities(conn, svcActivities, no_add=no_add)
+        self._global_logger.info(f"[SYNC PROCESS] Successfully got the list of activities to sync for hub user id {self.user['_id']} from partner {conn.Service.ID}")

```

Comment:
Not in Run function because of weird Exception management that allow to write "Success" even if the listing crashed